src/gui/interface/Header Files/datapage.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * @file datapage.h 
+ * @brief Interface for the data page
+ * @copyright Copyright (C) 2021  Andrew Ritchie
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef SRC_DATAPAGE_H_
+#define SRC_DATAPAGE_H_
+
+#include <QMainWindow>
+
+namespace Ui {
```

Comment:
Would UI look better?

Reply:
It probably would look better, but I guess `Ui` matches the "official" naming scheme

Reply:
I have used GUI now to fix this and other comments! :)